src/sentry/testutils/fixtures.py

```diff
@@ -721,6 +721,8 @@ def create_project_uptime_subscription(
     ) -> ProjectUptimeSubscription:
         if project is None:
             project = self.project
+        if env is None:
+            env = self.environment
```

Comment:
Do we always actually have an env? Not sure if we should make it impossible to set env to none

Reply:
it's a required field in the form. I think we always want them to set it